jupyterlab_nodeeditor/yggdrasil_support.py

```diff
@@ -1,9 +1,37 @@
 import uuid
-
 import yaml
 
 from .node_editor import NodeEditor
 
+from yggdrasil.examples import yamls as ex_yamls
+import jupyterlab_nodeeditor as jlne
+import yggdrasil.yamlfile
+import re
+
+photosynthesis_model = {}
+with open(ex_yamls['fakeplant']['python'], "r") as test_model:
+    photosynthesis_model = yaml.safe_load(test_model)['model']
+    
+# Hard coded function to clean up Photosynthesis model
+def dict_conversion(model_dict):
+    model_dict.pop("language")
+    model_dict.pop("args")
+    model_dict["title"] = model_dict.pop("name")
+    for i in model_dict['inputs']:
```

Comment:
Could we use a variable other than `i`?  I usually think of `i` as an idiom for an iteration counter, but here it's the actual object.  Maybe here it could be `inp` and below it could be `out`.